build.sh

```diff
@@ -12,18 +12,31 @@ echo "docker login done"
 export BUILD_ARGS="--rm=false --build-arg TAG=${CIRCLE_TAG} --build-arg SHA1=${CIRCLE_SHA1} --build-arg DATE=$(date +%F-%T)"
 echo  "BUILD_ARGS: $BUILD_ARGS"
 
-docker build -t "21zoo/redis_exporter:$CIRCLE_TAG" $BUILD_ARGS .
-docker push     "21zoo/redis_exporter:$CIRCLE_TAG"
-
-docker build -t "21zoo/redis_exporter:latest" $BUILD_ARGS .
-docker push     "21zoo/redis_exporter:latest"
-
-docker build -t "oliver006/redis_exporter:$CIRCLE_TAG" $BUILD_ARGS  .
-docker push     "oliver006/redis_exporter:$CIRCLE_TAG"
-
-docker build -t "oliver006/redis_exporter:latest" $BUILD_ARGS  .
-docker push     "oliver006/redis_exporter:latest"
-
+# Scratch image
+docker build --target scratch \
```

Comment:
`--target scratch` - this is neat, didn't know you can do that.

Reply:
Yes we can name build stages and reference them this way, also we can use `--from` to copy files from external images and not only build stages. Thank you for approving the request, BTW very nice work.

Reply:
Interesting, thanks for explaining. I clearly did not keep up with what you can do with multistage builds...